doc/maintainer-guide.md

```diff
@@ -0,0 +1,30 @@
+# Reviewing PRs
+
+Approving a PR means that the reviewer has deemed the PR ready to be merged.
+
+While reviewing
+
```

Comment:
I would add 2 points here:
 - Questions whether the new functionality is needed in the project and if it follows the vision of the projects ("Do we really want/need that feature?").
 - Questions whether the maintainer will be willing to maintain this feature on the long term ("Do we have the resources to maintain this feature?")